superset/migrations/versions/fb13d49b72f9_better_filters.py

```diff
@@ -0,0 +1,84 @@
+"""better_filters
+
+Revision ID: fb13d49b72f9
+Revises: 6c7537a6004a
+Create Date: 2018-12-11 22:03:21.612516
+
+"""
+import json
+import logging
+
+from alembic import op
+from sqlalchemy import Column, Integer, String, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+from superset import db
+
+# revision identifiers, used by Alembic.
+revision = 'fb13d49b72f9'
+down_revision = 'de021a1ca60d'
+
+Base = declarative_base()
+
+
+class Slice(Base):
+    __tablename__ = 'slices'
+
+    id = Column(Integer, primary_key=True)
+    params = Column(Text)
+    viz_type = Column(String(250))
+    slice_name = Column(String(250))
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = db.Session(bind=bind)
+
+    filter_box_slices = session.query(Slice).filter_by(viz_type='filter_box')
+    for slc in filter_box_slices.all():
+        try:
+            params = json.loads(slc.params)
+            logging.info(f'Upgrading {slc.slice_name}')
+            cols = params.get('groupby')
+            metrics = params.get('metrics')
```

Comment:
@mistercrunch it looks like the param for filter_box was `metric` so this isn't migrating any of the filter box metrics correctly, they are getting set to `null` cc: @graceguo-supercat

Reply:
You're right, and I'm confused as to how this got through... Let me send a fix now